service/src/main/java/bio/terra/workspace/service/folder/FolderService.java

```diff
@@ -105,7 +114,8 @@ private void getResourcesInFolder(
       UUID workspaceId,
```

Comment:
Can you add javadoc

/** Populates parameters controlledResources, referencedResources with resources in specified folder. */

I was confused because `getResourceInFolder` sounds like it returns resources. Maybe rename to `collectResourcesInFolder`.